macosLaunch.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+echo "Launching api rtmpserver and client terminals..."
+
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+
+osascript -e 'tell app "Terminal" to do script "cd '$DIR'/api/ && npm start"'
+
+osascript -e 'tell app "Terminal" to do script "cd '$DIR'/rtmpserver/ && npm start"'
+
+osascript -e 'tell app "Terminal" to do script "cd '$DIR'/client/ && npm start"'
```

Comment:
add a new line to the end of the file pls

Reply:
Update: Added new line at end of file